src/Config.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Netzstrategen\MultiStock;
+
+/**
+ * Plugin settings functionality.
+ */
+class Config {
+
+  /**
+   * Config JSON file.
+   *
+   * @var string
+   */
+  const CONFIG_FILE = '/config.json';
+
+  /**
+   * Cache expiration time.
+   *
+   * @var int
+   */
+  const CACHE_DURATION = 43200;
+
+  /**
+   * Returns the plugin configurations.
+   *
+   * @param string $key
+   *   The configuration JSON key to return.
+   *
+   * @return array
+   *   The plugin configurations.
+   */
+  public static function get($key = '') {
+    try {
+      $transient_id = Plugin::PREFIX . '_config';
+      if ($transient = get_transient($transient_id)) {
```

Comment:
The caching here is unnecessary, as reading and parsing a small JSON file from disk is equally fast under normal circumstances.